src/data/nat/bitwise.lean

```diff
@@ -104,6 +105,29 @@ begin
     linarith }
 end
 
+@[simp]
+lemma test_bit_two_pow_self {n : ℕ} : test_bit (2 ^ n) n = tt :=
```

Comment:
I may have said before that variables can be implicit if they're on both sides of an equality, but I've changed my mind on this; `simp` and `rw` don't really care unless you want to rewrite at a specific place, and then it's better to give an extra argument than to put in `@`. I think variables can still be made implicit if they appear on both sides of an `iff` though.
```suggestion
lemma test_bit_two_pow_self (n : ℕ) : test_bit (2 ^ n) n = tt :=
```